tests/Dockerfile

```diff
@@ -1,5 +1,7 @@
-# syntax = docker/dockerfile:1.4
+# syntax=docker/dockerfile:1
```

Comment:
Note that the whitespace is allowed in these; I'm somewhat trying to use a consistent formatting in our docs and repositories (`#<space>syntax=<syntax>`); consistency helps with searching for occurrences, as well as might avoid questions from casual visitors (why does the documentation show X, and here it shows X?)